.github/workflows/test-deploy-wf-service.yml

```
@@ -36,70 +36,231 @@
+      - name: Checkout wf-data-migration
+        uses: actions/checkout@v4
+        with:
+          repository: ballerine-io/wf-data-migration
+          token: ${{ secrets.SUBMODULES_TOKEN }}
+          ref: ${{ inputs.environment }}
+          fetch-depth: 1
+          path: services/workflows-service/prisma/data-migrations
+      
+      - name: Cache Docker layers
+        id: cache
+        uses: actions/cache@v2
```

actionlint

the runner of "actions/cache@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)